shared/src/test/diff/nu/ModuleParameters.mls

```diff
@@ -10,21 +10,15 @@ module A(x: int) { fun y = x }
 //│ module A(x: int) {
 //│   fun y: int
 //│ }
-//│ Code generation encountered an error:
-//│   unresolved symbol x
 
-:re
 A.x
 //│ int
 //│ res
-//│ Runtime error:
-//│   ReferenceError: A is not defined
+//│     = undefined
```

Comment:
Why does it return `undefined`? Shouldn't it just access `this.#x`?

Reply:
Oh never mind this one is okay. But there are other unexplained `undefined`s in this PR.

Reply:
Accessing `this.#x` is still under progress